extensions/wikia/Recirculation/RecirculationController.class.php

```diff
@@ -20,6 +21,7 @@ public function index() {
 		$posts = $fandomDataService->getPosts( $this->type );
 
 		if ( count( $posts ) > 0 ) {
+			$this->response->setCacheValidity( self::CACHE_DURATION );
```

Comment:
5 minutes does seem too short and 3 hours too long, however what could you possibly name a standard in between `SHORT` and `VERY_SHORT`? I went with just using `WikiaResponse::CACHE_VERY_SHORT` for now. Although, I'm not totally opposed to the idea of using `WikiaResponse::CACHE_SHORT` if we think 5 minutes isn't enough time